package.json

```diff
@@ -48,7 +48,7 @@
     "react-router": "^5.2.0",
     "react-router-dom": "^5.2.0",
     "react-stately": "^3.9.0",
-    "react-virtuoso": "^1.9.0",
+    "react-virtuoso": "^2.3.1",
```

Comment:
**question:** Any neat perf updated with the major version change?

Reply:
No, but the update provided the ability to use their logging / debugging, which was nice.